kclosest.py

```diff
@@ -0,0 +1,20 @@
+#Brute force solution to solve the problem
+#Time Complexity: O(nlogn)
+#Space complexity: O(2n)
+arr = [1,2,3,4,5]
+k = 4
+x = 3
+li = list()
+li2 = list()
+def findClose(arr,k,x):
```

Comment:
This code does not pass all the testcases over on LeetCode, Please rework your logic try the heap solution as well.
https://leetcode.com/problems/find-k-closest-elements/